### PR TITLE
consumed the api for each school homepage and the list of quizes for the select quiz section

### PR DESCRIPTION
**Pull request by silasade**

### CODE REVIEW

File: package.json
Comment (a-k-o-r-e-d-e) [inline package.json:1]: you need two cookie packages?

File: src/Global Components/Pagininate/Paginate1.tsx
Comment (a-k-o-r-e-d-e) [inline src/Global Components/Pagininate/Paginate1.tsx:1]: whats Paginate1.tsx ? does that suggest there will be a Paginate2.tsx

Proper file and variable names are required

Comment (a-k-o-r-e-d-e) [inline src/Global Components/Pagininate/Paginate1.tsx:1]: PaginatedItems1?? 

Nah, nah, i want proper file, variable and Class Names

**Review by a-k-o-r-e-d-e — CHANGES_REQUESTED** [review]
Also create a central point or file for your types and declare them there. so they can be reused

File: package.json
Comment (silasade) [reply package.json:1]: Uninstalled JS-COOKIE

File: src/Global Components/Pagininate/Paginate1.tsx
Comment (silasade) [reply src/Global Components/Pagininate/Paginate1.tsx:1]: Fixed the name.

Comment (silasade) [reply src/Global Components/Pagininate/Paginate1.tsx:1]: It has been renamed

**Review by a-k-o-r-e-d-e — APPROVED** [review]

**Pull request merged**